setup.py

```diff
@@ -33,6 +33,7 @@
         "rdflib-jsonld==0.4.0",
         "requests>=2.23,<3.0",
         "uritemplate>=3.0,<4.0",
+        "rfc3987==1.3.8",
```

Comment:
ideally we can keep these in alphabetical order for clarity. not a huge deal though